experimenter/tests/integration/nimbus/test_mobile_first_run.py

```diff
@@ -0,0 +1,175 @@
+from urllib.parse import urljoin
+
+import pytest
+
+from nimbus.pages.experimenter.summary import SummaryPage
+from nimbus.utils import helpers
+
+
+@pytest.mark.mobile_first_run
+def test_first_run_release_date(
+    base_url,
+    selenium,
+    kinto_client,
+    slugify,
+    experiment_name,
+):
+    application = "FENIX"
+    experiment_slug = str(slugify(experiment_name))
+    targeting = helpers.load_targeting_configs(app=application)[0]
+    data = {
+        "hypothesis": "Test Hypothesis",
+        "application": application,
+        "changelogMessage": "test updates",
+        "targetingConfigSlug": targeting,
+        "publicDescription": "Some sort of Fancy Words",
+        "riskRevenue": False,
+        "riskPartnerRelated": False,
+        "riskBrand": False,
+        "featureConfigIds": [2],
+        "referenceBranch": {
+            "description": "reference branch",
+            "name": "Branch 1",
+            "ratio": 50,
+            "featureValues": [
+                {
+                    "featureConfig": "1",
+                    "value": "{}",
+                },
+            ],
+        },
+        "treatmentBranches": [],
+        "populationPercent": "100",
+        "totalEnrolledClients": 55,
+        "isFirstRun": True,
+        "proposedReleaseDate": "2023-12-12",
+    }
+    helpers.create_experiment(
+        experiment_slug,
+        app=application,
+        targeting=targeting,
+        data=data,
+    )
+
+    summary = SummaryPage(selenium, urljoin(base_url, experiment_slug)).open()
+    summary.launch_and_approve()
+
+    kinto_client.approve()
+
+    summary = SummaryPage(selenium, urljoin(base_url, experiment_slug)).open()
+    summary.wait_for_live_status()
+
+    assert True, summary.first_run
+    assert "2023-12-12", summary.proposed_release_date
```

Comment:
What is happening in these assert statements?  `True` and non empty string are truthy so these will always pass?